packages/fbjs/src/core/dom/isNode.js

```diff
@@ -15,8 +15,10 @@
  * @return {boolean} Whether or not the object is a DOM node.
  */
 function isNode(object) {
+  var doc = object ? (object.ownerDocument || object) : document;
+  var defaultView = doc.defaultView || window;
```

Comment:
I’m thinking `doc` can’t be null. It definitely passes the `isNode` tests.

If `object` is null, `doc` is `document`
If `object` is not null but `object.ownerDocument` is null, `doc` becomes `object`, which wasn’t null
If `object` is not null and `object.ownerDocument` is not null, it won’t throw